canonical-web-and-design/ubuntu.com
# Replace /gov section and update gov page

## Done

- Replaced the `/gov` section, files found [here](https://github.com/steverydz/ubuntu.com/tree/e8b7a07e86acba3012160fb805dd0b949056adb7/templates/gov)
- Updated the `/gov/index.html` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/gov
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [copy doc ](https://docs.google.com/document/d/1umM1XMEzqeXklPkyCjDr98Q-RvwCcNNXgwNwfLoynTo/edit#)


## Issue / Card

Fixes #9378
